client/verta/tests/unit_tests/deployment/test_build.py

```diff
@@ -92,3 +92,20 @@ def test_model_version_list_builds(
         sorted(build_dicts, key=lambda d: d["id"]),
     ):
         assert_build_fields(build, build_dict)
+
+
+@given(build_dict=build_dict())
+def test_build_start_scan_not_external_exception(mock_conn, build_dict):
+    """Test that start_scan() raises the expected error when the `external` parameter is not
+    included or False.  This can be removed when we support internal scans.
+    """
+    build = Build(mock_conn, build_dict)
+    with pytest.raises(
+        NotImplementedError,
+        match=(
+            "internal scans are not yet supported.  Please use `external=True`"
+            " parameter."
+        ),
+    ):
+        build.start_scan()
+        build.start_scan(external=False)
```

Comment:
This isn't going to work. `build.start_scan()` is going to raise an error and the `with` will exit, so `build.start_scan(external=False)` will never get run.

Reply:
Doh.  Fixed it with separate context managers for each condition.

Reply:
@liuverta I'll leave this to you to resolve if you are good with the solution.

Reply:
This is moot now that `external` is a required param.  I dropped check for `start_scan()` with no params